src/main/java/com/teragrep/rlp_03/SocketPoll.java

```diff
@@ -94,31 +93,25 @@ public SocketPoll(
     public void poll() throws IOException {
         int readyKeys = selector.select();
 
-        LOGGER.debug("readyKeys <{}>",readyKeys);
+        LOGGER.debug("readyKeys <{}>", readyKeys);
 
         Set<SelectionKey> selectionKeys = selector.selectedKeys();
         LOGGER.debug("selectionKeys <{}> ", selectionKeys);
         for (SelectionKey selectionKey : selectionKeys) {
             if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug(
-                        "selectionKey <{}>: " +
-                                "isValid <{}>, " +
-                                "isConnectable <{}>, " +
-                                "isAcceptable <{}>, " +
-                                "isReadable <{}>, " +
-                                "isWritable <{}>",
-                        selectionKey,
-                        selectionKey.isValid(),
-                        selectionKey.isConnectable(),
-                        selectionKey.isAcceptable(),
-                        selectionKey.isReadable(),
-                        selectionKey.isWritable()
-                );
+                LOGGER
+                        .debug(
```

Comment:
Perhaps these logs with bunch of variables could be replaced with:
https://www.baeldung.com/slf4j-parameterized-logging
See "addKeyValue() approach" on section "5. Fluent logging"

Reply:
Think it would be a cleaner approach if possible.